### PR TITLE
Updated deb-tool README

### PR DESCRIPTION
**Pull request by svick**
This link was fixed in cli's copy of this document in https://github.com/dotnet/cli/pull/3472. (Not sure why there are two copies of it.)

Also improved formatting of the README.

### CODE REVIEW

File: tools/dotnet-deb-tool/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ directory structure with your desired files to be included.
 Note: The default install root is `/usr/share/{package_name}` where package_name is replaced with the name of the created package
 
 ## full example debian_config.json
-Note: Use the commentless version here (https://github.com/dotnet/cli/blob/master/packaging/debian/package_tool/example_config.json)
+Note: Use the commentless version [here](example_config.json).
 
 ```json
```
Comment (jasonwilliams200OK) [inline tools/dotnet-deb-tool/README.md]: nit: changing the language from `json` to `js` or `javascript` will render the comment color correctly (JSON std doesn't support C-style comments, GitHub and JavaScript parsers in browsers and node.js follow the std., however, JSON.NET is forgiving):

With ```` ```json ````:
```json
{
  "a": "b" // test comment
}
```

With ```` ```js ```` or ```` ```javascript ````:
```js
{
  "a": "b" // test comment
}
```

as JSON code is a valid JavaScript, but not vice-versa.

Comment (svick) [inline tools/dotnet-deb-tool/README.md]: @jasonwilliams200OK That's a good idea, done. Thanks.

**Comment by brthor** [comment]
The version in cli has since been removed. 

LGTM :shipit:

**Pull request merged**